pkg/browser/detect.go

```diff
@@ -100,40 +114,46 @@ func Find() (string, error) {
 }
 
 func GetBrowserKey(b string) string {
-	if strings.Contains(strings.ToLower(b), "chrome") {
+	if strings.EqualFold(b, ChromeKey) {
 		return ChromeKey
 	}
 
-	if strings.ToLower(b) == "firefox developer edition" {
+	if strings.EqualFold(b, FirefoxDevEditionKey) {
 		return FirefoxDevEditionKey
 	}
 
-	if strings.ToLower(b) == "firefox nightly" {
+	if strings.EqualFold(b, FirefoxNightlyKey) {
 		return FirefoxNightlyKey
 	}
-
-	if strings.Contains(strings.ToLower(b), "brave") {
+	if strings.HasPrefix(strings.ToLower(b), "flatpak") {
```

Comment:
This was placed higher up the list since it was conflicting with the "mozilla" check.  

Looking again, I think this function needs a small refactor, since keys are upper cased and separated with underscores, whereas `b` is regular space-separated text.